packages/create-farm/index.ts

```diff
@@ -160,6 +160,11 @@ function writePackageJson(dest: string, options: IResultType) {
 
   pkg.name = options.projectName;
 
+  if (isYarnInstalled) {
```

Comment:
`options.packageManager` should be used here, make sure this logic executed only when the user chooses `yarn` as pacakge manager. And `pacakgeManager` should be passed as arguments in https://github.com/farm-fe/farm/pull/985/files#diff-53f16bc3fa79da2028e77129b2f7ad141dac3fb81fcd74a7e1e44f8cf5aff945R129

Reply:
When `options.packageManager` is directly used, it doesn't accurately identify `yarn` in the `yarn create farm` command.

To resolve this, I've used the same strategy as `installationDeps` by utilizing `currentPkgManager`.

This ensures alignment with the prompt and also properly recognizes `yarn`. 

Could you please share your thoughts on this?

Reply:
Thanks, that's exactly what i think